modules/core/src/main/scala/org/scalasteward/core/io/package.scala

```diff
@@ -48,4 +50,9 @@ package object io {
   private def isScalafmtCoreUpdate(update: Update): Boolean =
     update.groupId === GroupId("org.scalameta") &&
       update.artifactIds.exists(_.name === "scalafmt-core")
+
+  def readResource[F[_]](resource: String)(implicit F: Sync[F]): F[String] =
+    Resource
+      .fromAutoCloseable(F.delay(Source.fromResource(resource)))
+      .use(src => F.delay(src.mkString))
```

Comment:
I just noticed this will block (maybe `fromResource` too)... shall we add Blocker here?